ws-tests/test_v2_find_trees.py

```diff
@@ -0,0 +1,26 @@
+import sys
+from check import *
+from check_oti import *
+status = 0
+
+check_v2_tree = check_blob([field(u'oti_tree_id', check_string),
+                            field(u'ot:studyId', check_string),
+                            field(u'is_deprecated', check_boolean),  # never true
+                            field(u'ot:originalLabel', check_string),  # useless
+                            field(u'ot:ottId', check_integer),  # useless
+                            field(u'ot:ottTaxonName', check_string),  # useless
+                            field(u'ot:branchLengthMode', check_string),
+                            field(u'ot:branchLengthDescription', check_string)])
+
+check_v2_study = check_blob(study_fields +
+                            [field(u'is_deprecated', check_boolean),
+                             field(u'matched_trees', check_list(check_v2_tree))])
+
+status += \
```

Comment:
This kinda stopped my brain for a minute, until I realized it was just a line continuation.